### ISSUE TITLE
Which files or folders to back up to properly restore MegaSync after installation of OS?

### ISSUE THREAD

**Issue by sojusnik** [meganz/MEGAsync#900]
Which folders or files have to be backed up, in order to properly restore the same config of MegaSync after a fresh installation of the OS, in this case Ubuntu?

Previously it was enough to back up and restore only `~/.local/share/data/Mega Limited/MEGAsync/MEGAsync.cfg`, but nowadays not any more, because you still have to set up the sync and backup folders even after the .cfg file was copied over and MegaSync launched for the first time.

**Comment by rautamiekka** [comment]
Did you try taking the whole `~/.local/share/data/Mega Limited/` tree ? I haven't tested.

**Comment by sojusnik** [comment]
No, because the `megaclient_statecache…` files under `~/.local/share/data/Mega Limited/MEGAsync` seem like cache files to me, which usually shouldn't be backuped and restored.

**Comment by rautamiekka** [comment]
Don't knock before you try it, or read the source code.
